README.md

```diff
@@ -30,32 +30,57 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+
+A forEach() method executes a provided function once for each array while map() creates a new array with the results of calling a provided function on every element in the calling array.
+
+
+
 2. What is the difference between a function and a method?
 
+
+A method is a function when an object is associated it where as a function is a function when no object is associated with it.
+
+
+
 3. What is closure?
 
+
+A Closure is a nested function with the ability to reference things (variables) outside its normal scope.
```

Comment:
Closure is a concept that allows  for reaching out of scope to use what it needs